Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2180,18 +2180,12 @@ extension BrowserViewController {
         present(dBOnboardingViewController, animated: true, completion: nil)
     }
 
-    @discardableResult func presentUpdateViewController(_ force: Bool = false, animated: Bool = true) -> Bool {
-        let cleanInstall = UpdateViewModel.isCleanInstall(userPrefs: profile.prefs)
-        let coverSheetSupportedAppVersion = UpdateViewModel.coverSheetSupportedAppVersion
-        if force || UpdateViewModel.shouldShowUpdateSheet(userPrefs: profile.prefs, isCleanInstall: cleanInstall, supportedAppVersions: coverSheetSupportedAppVersion) {
-            let updateViewController = UpdateViewController()
-
-            updateViewController.viewModel.startBrowsing = {
-                updateViewController.dismiss(animated: true) {
-                    if self.navigationController?.viewControllers.count ?? 0 > 1 {
-                        _ = self.navigationController?.popToRootViewController(animated: true)
-                    }
-                }
+    func presentUpdateViewController(_ force: Bool = false, animated: Bool = true) {
```

Comment:
nit: `force` seems weird to me and doesn't read nicley. We could update to `shouldForcePresentation` and it would read better.